Samples/BlazorExample/BlazorExample/Server/Startup.cs

```diff
@@ -23,7 +24,14 @@ public void ConfigureServices(IServiceCollection services)
                   new[] { "application/octet-stream" });
       });
 
-      services.AddTransient(typeof(DataAccess.IPersonDal), typeof(DataAccess.PersonDal));
+      //services.AddTransient(typeof(DataAccess.IPersonDal), typeof(DataAccess.PersonDal));
```

Comment:
Is there a generic `AddTransient`? Like `AddTransient<DataAccess.IPersonDal, DataAccess.Mock.PersonDal>()`?

Reply:
I will try